media-gfx/librecad/librecad-2.0.7.ebuild

```diff
@@ -12,8 +12,8 @@ SRC_URI="https://github.com/LibreCAD/LibreCAD/archive/${PV/_/}.zip -> ${P}.zip"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
You are right, I didn't knew they were for prefix, I couldn't find info after a google search, and tought they were redundant, but I was just reading the arch.list file (I'm still learning about writing ebuilds).
Do I send it again without that?